Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -323,7 +323,7 @@ def audit_file
       end
       valid_alias_names = [alias_name_major, alias_name_major_minor]
 
-      if formula.tap
+      unless ["homebrew/core", ""].include? formula.tap.to_s
```

Comment:
I think the version suggested in the PR was easier to follow but I'd suggest using `formula.tap.core_tap?` instead.